smac/tae/execute_ta_run_old.py

```diff
@@ -94,16 +84,18 @@ def run(self, config, instance=None,
                 quality = float(quality)
                 seed = int(seed)
 
-        if status in ["SAT", "UNSAT", "SUCCESS"]:
+        if status.upper() in ["SAT", "UNSAT", "SUCCESS"]:
             status = StatusType.SUCCESS
-        elif status in ["TIMEOUT"]:
+        elif status.upper() in ["TIMEOUT"]:
             status = StatusType.TIMEOUT
-        elif status in ["CRASHED"]:
+        elif status.upper() in ["CRASHED"]:
             status = StatusType.CRASHED
-        elif status in ["ABORT"]:
+        elif status.upper() in ["ABORT"]:
             status = StatusType.ABORT
-        elif status in ["MEMOUT"]:
+        elif status.upper() in ["MEMOUT"]:
             status = StatusType.MEMOUT
+        else:
+            status = StatusType.CRASHED
 
```

Comment:
I'm not sure if we want the status to be crashed if the output of the target algorithm is unintelligible.

Reply:
Thanks for pointing this out; added warn logging message.